jit/v1/jit_v1.2.0-Beta-2.yaml

```diff
@@ -0,0 +1,1312 @@
+          x-stoplight:
+            id: 8mcuorc4u21bd
+        vesselPosition:
+          $ref: '#/components/schemas/GeoLocation'
+          x-stoplight:
+            id: dd3eo8z0qppt0
+        milesToDestinationPort:
+          type: number
+          x-stoplight:
+            id: 2ny5o39r98ybz
+          description: Remaining distance reported by the vessel to the next destination port in nautical miles.
+          example: 245.45
+          format: float
+        vesselDraft:
+          type: number
+          x-stoplight:
+            id: ul9ocnas3pqy0
+          description: "The actual draft of the vessel.\r\n\r\nIf the draft is specified in feet (FOT) then the decimal part should be concidered as a fraction of a foot and not as a number of inches. E.g. 120.5 feet means 120 and a half foot (which would be 120'6\")."
+          format: float
+          example: 12.5
+        vesselDraftUnit:
+          type: string
+          x-stoplight:
+            id: 5w9wxka5cnoti
+          example: MTR
+          description: "The unit of measure which can be expressed in\r\n\r\n- MTR (Meter)\r\n- FOT (Foot)"
+    TransportCall:
+      title: TransportCall
+      x-stoplight:
+        id: ui3am52u5xhzr
+      type: object
+      required:
+        - modeOfTransport
+      properties:
+        transportCallID:
+          type: string
+          x-stoplight:
+            id: xoqb3ghweft1v
+          example: 123e4567-e89b-12d3-a456-426614174000
+          maxLength: 100
+          description: "The unique identifier for a transport call\r\n\r\nDeprecated: Use transportCallReference instead."
+        transportCallReference:
+          type: string
+          x-stoplight:
+            id: xyxxa47omn33y
+          description: "A carrier definied reference to a TransportCall.\r\n\r\nIn the case the Means of Transport is a Vessel and the facility is a Port/Terminal - this reference should be considered a Terminal Call Reference."
+          maxLength: 100
+        portVisitReference:
+          type: string
+          x-stoplight:
+            id: cnwnue5t6863v
+          example: NLRTM1234589
+          description: The unique reference that can be used to link different transportCallReferences to the same port visit. The reference is provided by the port to uniquely identify a port call.
+          maxLength: 50
+        carrierServiceCode:
+          type: string
+          x-stoplight:
+            id: 29bvr3ii3ucdl
+          description: The Carrier specific code of the service for which the schedule details are published.
+          example: FE1
+          maxLength: 5
+        carrierVoyageNumber:
+          type: string
+          x-stoplight:
+            id: qgbc1iwalz2iv
+          deprecated: true
+          example: 2103S
+          maxLength: 50
+          description: "The vessel operator-specific identifier of the Voyage.\r\nIn case there are multiple voyages the export voyage is chosen."
```

Comment:
```suggestion
          description: "The vessel operator-specific identifier of the Voyage.\r\nIn case there are multiple voyages the export voyage is chosen.\r\n**Deprecated**: Use `carrierExportVoyageNumber` or `carrierImportVoyageNumber` instead."
```

Reply:
I know the above was not part of the original description - but want to include it. Feel free to reject it - but then please make an issue to update afterwards ;-)

Reply:
Added this as we will be publishing JIT 2.0 with all the changes in 1.2.